guide/payments/contacts.md

```diff
@@ -0,0 +1,229 @@
+    retina = "/assets/images/guide/payments/contacts/contacts-header@2x.jpg"
+    mobile = "/assets/images/guide/payments/contacts/contacts-header-mobile.jpg"
+    mobileRetina = "/assets/images/guide/payments/contacts/contacts-header-mobile@2x.jpg"
+    alt-text = "Illustration of large contact cards"
+    width = 1600
+    height = 400
+    layout = "full-width"
+%}
+
+
+# Contacts
+{:.no_toc}
+
+<!--
+
+
+-->
+
+Whether we’re sending emails, physical mail, or following someone on social media, we primarily think in terms of names and faces, and not the respective address or user ID.
+
+Invoices, node IDs and other transaction endpoints in bitcoin and Lightning are highly unintuitive. Abstracting them via a contact list can create a much smoother user experience. There are many [payment request formats]({{ '/guide/payments/request/payment-request-formats/' | relative_url }}), each with unique properties and varying levels of maturity and adoption, requiring unique design solutions. This page uses the more approachable term "address", along with various UI techniques, to abstract these complexities for users.
+
+Let's go over common user interactions around contacts. This will illustrate how such a feature could work, and helps explain the underlying design problems and decisions.
+
+---
+
+<div class="glossary-toc" markdown="1">
+* Table of contents
+{:toc}
+</div>
+
+---
+
+### Adding a contact
+
+The most basic interaction is that a user manually adds a contact by entering their name and an address. This is likely not the most common user flow, as most contacts will be added to incoming payment requests, as illustrated further below. But wallets should generally support manual options, as users may not be able to avoid them.
+
+{% include image-gallery.html pages = page.imagesAddContact %}
+
+### Supporting various payment request formats
+
+Based on your use case, your application may choose to only support certain payment request formats. It should still recognize all formats, and provide the user with appropriate feedback. You may also choose to add warnings before permitting use of certain formats if there are privacy and security risks the user should be aware of.
+
+{% include image-gallery.html pages = page.imagesSupportedFormats %}
+
+### Contact editing
+
+The contacts screen should offer various features for editing and organization. Also see [Requesting bitcoin &middot; Contact cards]({{ '/guide/payments/request/#contact-cards' | relative_url }}).
+
+{% include image-gallery.html pages = page.imagesMultiAddressContact %}
+
+### Importing an address
+
+In this scenario, the user has copied a Lightning address to the clipboard and opened the app. The clipboard is automatically scanned, the app identifies the addres and offers the user appropriate options. Here, the user adds the address as a new contact.
+
+{% include image-gallery.html pages = page.imagesImportAddress %}
```

Comment:
The add contact button should prompt if the user wants to add to an existing contact or create a new contact.

Reply:
Good point. [Daniel had some feedback](https://www.figma.com/file/VB3GQdAnhl8yta44DY3PSV/Bitcoin-UI-Kit?node-id=3202%3A83932) on changing this modal to just show three buttons. This would simplify the overall effort by re-using existing flows (which would address your point, as the contacts modal has a "+" button).

Reply:
LGTM :)